src/test/java/uk/gov/hmcts/reform/civil/handler/callback/camunda/notification/ClaimantResponseConfirmsToProceedLiPRespondentNotificationHandlerTest.java

```diff
@@ -72,6 +73,50 @@ void shouldNotifyLipRespondent_whenInvoked() {
             );
         }
 
+        @Test
+        void shouldNotifyLipRespondent_whenTranslatedDocUploaded() {
+            CaseData caseData = CaseDataBuilder.builder().atStateClaimDetailsNotified().build();
+            caseData.setClaimantBilingualLanguagePreference("BOTH");
+            CallbackParams params = CallbackParamsBuilder.builder().of(ABOUT_TO_SUBMIT, caseData).request(
+                CallbackRequest.builder().eventId(CaseEvent.NOTIFY_LIP_RESPONDENT_CLAIMANT_CONFIRM_TO_PROCEED_TRANSLATED_DOC.name())
+                    .build()).build();
+
+            handler.handle(params);
+
+            verify(notificationService).sendMail(
```

Comment:
Suggestion:- Add times(1) so that it check exactly 1 call

Reply:
Done